### ISSUE TITLE
Remarks on documentation for minimal_scheduling

### ISSUE THREAD

**Issue by awischnewski** [ros-realtime/ros2-realtime-examples#6]
I have been going through some of the examples now and wanted to share the points which I think could be helpful for new users. I'll put this feedback on a per package basis and will start working on a pull request with some suggestions to implement it. If you have any comments on the points below or if I missed something, please let me know. 

- [x] Add example to use `chrt --fifo 89` prior to `ros2 run` to launch code instead of setting priorities within the code 
- [x] Its not clear to me, why you mention that the style of launching with the middleware example is helpful in cases where the middleware should *not* participate in the RT execution. Shouldn't it be the other way round? 
- [ ] What could be a good reason to use the spin_thread version instead of the main version? This makes sense in the callback_group example, but I couldn't think of an obvious one for the single thread version. This would be helpful for sure for a new user. For now, I would add a remark that this is basically a way to move the executor to a separate thread.

**Comment by awischnewski** [comment]
Just realized that @carlossvg already covered some of those points in the latest `rolling`. 

@carlossvg could you add some of your thoughts on the spin_thread version?

**Comment by carlossvg** [comment]
First off, thanks for the comments. 

>@carlossvg could you add some of your thoughts on the spin_thread version?

I created the spin_thread example just to show another way to configure the scheduling policy of a std::thread by getting the native handle. I added later the callback group example so now there is some overlapping. 

I don't see that much value in having a specific example to show this so I would consider removing it. I'm also considering merging the main_thread example and the middleware_threads one into one single example. I would add a command-line option to select if the scheduling policy should be set before or after the node creation. 

> Add example to use chrt --fifo 89 prior to ros2 run to launch code instead of setting priorities within the code
 
Yes, I think it's worth briefly mentioning it. I was thinking to do the same using taskset for the affinity example.
